src/Angor/Shared/Angor.Shared.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Blockcore.Core" Version="1.1.41" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="8.0.0" />
```

Comment:
why do we need `Hosting.Abstractions`? what component uses it?
does it mean we pull more data on load?

Reply:
This is related to the previous code and should be removed